Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/usr/local/lib/python3.9/dist-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/local/lib/python3.9/dist-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/local/lib/python3.9/dist-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/usr/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/usr/local/lib/python3.9/dist-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/usr/local/lib/python3.9/dist-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/usr/local/lib/python3.9/dist-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from sklearn.feature_extraction.text import TfidfVectorizer
# )
tfidf_vectorizer = TfidfVectorizer()
tfidf_separate = tfidf_vectorizer.fit_transform(text)

df_tfidf = pd.DataFrame(
    tfidf_separate.toarray(), columns=tfidf_vectorizer.get_feature_names(), index=df.index
)
X = df_tfidf.values
df_tfidf
------------------

[0;31m[0m
[0;31mAttributeError[0mTraceback (most recent call last)
[0;32m<ipython-input-4-edf9054314d2>[0m in [0;36m<cell line: 6>[0;34m()[0m
[1;32m      5[0m [0;34m[0m[0m
[1;32m      6[0m df_tfidf = pd.DataFrame(
[0;32m----> 7[0;31m     [0mtfidf_separate[0m[0;34m.[0m[0mtoarray[0m[0;34m([0m[0;34m)[0m[0;34m,[0m [0mcolumns[0m[0;34m=[0m[0mtfidf_vectorizer[0m[0;34m.[0m[0mget_feature_names[0m[0;34m([0m[0;34m)[0m[0;34m,[0m [0mindex[0m[0;34m=[0m[0mdf[0m[0;34m.[0m[0mindex[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      8[0m )
[1;32m      9[0m [0mX[0m [0;34m=[0m [0mdf_tfidf[0m[0;34m.[0m[0mvalues[0m[0;34m[0m[0;34m[0m[0m

[0;31mAttributeError[0m: 'TfidfVectorizer' object has no attribute 'get_feature_names'
AttributeError: 'TfidfVectorizer' object has no attribute 'get_feature_names'

